File: PYTHON/PowerSupply.py
from Komponen import Komponen

class PowerSupply(Komponen):
    def __init__(self, watt=0, merk="", nama=""):
        super().__init__(merk, nama)
        self.watt = watt

    def set_wattage(self, watt):
        self.watt = watt

    def get_wattage(self):
        return self.watt

File: JAVA/PowerSupply.java
public class PowerSupply extends Komponen {
    private int watt;

    public PowerSupply() {}

    public PowerSupply(int watt, String merk, String nama) {
        super(merk, nama);
        this.watt = watt;
    }

    public void setWattage(int watt) {
        this.watt = watt;
    }

    public int getWattage() {
        return watt;
    }
}
